library/Main.java

```diff
@@ -53,7 +54,27 @@ private static String Get_menu() {
 		  .append("  T  : increment date\n")
 		  .append("  Q  : quit\n")
 		  .append("\n")
-		  .append("Choice : ");
+		  .append("Choice : ");*/
+		
+		  sb.append("  M  : add member\n")

```

Comment:
If you wanna pout sb.append then there would be semicolon in the end of each line.